FinnUI/Sources/Components/SaveSearchView/SaveSearchView.swift

```diff
@@ -47,7 +47,7 @@ public class SaveSearchView: UIView {
     private lazy var searchNameLabel: Label = {
         let label = Label(style: .title3Strong, withAutoLayout: true)
         label.textAlignment = .center
-        label.numberOfLines = 0
+        label.numberOfLines = 2
```

Comment:
I'm not sure if we should set this one🤔 It's up to the user if he/she wants a long name for the search, and I assume the owners of iPhone SEs are used to not seing all the content all of the time😄

Reply:
I kinda agree, it should be possible to see the full name somewhere 😅  Will remove it. Can't always do what's best for small screen users 😂